diff --git a/trustm_lib/optiga/include/optiga/optiga_lib_config_m_v3.h b/trustm_lib/optiga/include/optiga/optiga_lib_config_m_v3.h
index 72039eb..b1fdeac 100644
--- a/trustm_lib/optiga/include/optiga/optiga_lib_config_m_v3.h
+++ b/trustm_lib/optiga/include/optiga/optiga_lib_config_m_v3.h
@@ -110,7 +110,7 @@ extern "C" {
      *         Warm Reset - (2) : This is applicable if the host platform doesn't have GPIO option for VDD. \n
      *         Any other value will lead to error
      */
-    #define OPTIGA_COMMS_DEFAULT_RESET_TYPE     (0U)
+    #define OPTIGA_COMMS_DEFAULT_RESET_TYPE     (1U)
     
     /** @brief NULL parameter check.
      *         To disable the check, undefine the macro
diff --git a/trustm_lib/pal/linux/target/rpi3/pal_ifx_i2c_config.c b/trustm_lib/pal/linux/target/rpi3/pal_ifx_i2c_config.c
index fc5220a..7c1e78a 100644
--- a/trustm_lib/pal/linux/target/rpi3/pal_ifx_i2c_config.c
+++ b/trustm_lib/pal/linux/target/rpi3/pal_ifx_i2c_config.c
@@ -61,8 +61,6 @@ pal_i2c_t optiga_pal_i2c_context_0 =
     0x30
 };
 
-static struct pal_linux_gpio pin_reset = {GPIO_PIN_RESET, -1};
-static struct pal_linux_gpio pin_vdd = {GPIO_PIN_VDD, -1};
 
 /**
 * \brief PAL vdd pin configuration for OPTIGA. 
@@ -70,7 +68,7 @@ static struct pal_linux_gpio pin_vdd = {GPIO_PIN_VDD, -1};
 pal_gpio_t optiga_vdd_0 =
 {
     // Platform specific GPIO context for the pin used to toggle Vdd.
-    (void*)&pin_vdd
+    NULL
 };
 
 /**
@@ -79,7 +77,7 @@ pal_gpio_t optiga_vdd_0 =
 pal_gpio_t optiga_reset_0 =
 {
     // Platform specific GPIO context for the pin used to toggle Reset.
-    (void*)&pin_reset
+    NULL
 };
 
 /**
